### ISSUE TITLE
Geode Is installed for GD v2.206, but didn't seem to load at all while launching

### ISSUE THREAD

**Issue by JoseInATux** [geode-sdk/geode#825]
### Geode Issue

- [X] I confirm that this bug is NOT related to a mod but directly to Geode Loader itself.

### Platform

Windows

### Geode Version

v2.0.0-beta.27

### Mods Installed

Mods are Installed but weren't causing any issues so far.

### Expected Behavior

- Regularly it supposed to show what the modloader was doing (the Mods installed and loaded).
- Shows the Button for the Geode Mods Menu (IMG 1)
1: ![2f4jpn6w](https://github.com/geode-sdk/geode/assets/79761367/de4935a1-166f-415f-a858-12a4df339779) (The GD main menu with the Geode icon at the bottom)

### Actual Behavior

As the game was loading, no indication of the Geode modloader were present (mods loaded) (IMG 1), neither the geode button was present aswell (IMG 2).
1: ![Screenshot 2024-06-01 193734](https://github.com/geode-sdk/geode/assets/79761367/c1358719-f1af-445d-8884-807b5f7ef486)
2: ![Screenshot 2024-06-01 193746](https://github.com/geode-sdk/geode/assets/79761367/b46c6554-3bc0-4113-aec8-2d70e1fdc31a)

### Steps to Reproduce

Basically everytime I launch GD, it happens.

### Additional Information

I have tried updating to latest version (Geode Version), I have tried reinstalling the Modloader, and i made sure the files where there (IMG 1).
I assume GD V2.206, or FMod, is the likely culprit of this issue.

1: 
![Screenshot 2024-06-01 193318](https://github.com/geode-sdk/geode/assets/79761367/f7cc3718-fc64-4e7d-a166-19786fb4919b)

**Comment by kernaltrap8** [comment]
yeah same issue here. my guess is Geode needs to be updated since it probably depends on hooking stuff in GeometryDash.exe (makes sense since its a game mod/framework) and since the game was updated, its going to have new offsets since it was recompiled

**Comment by kernaltrap8** [comment]
![image](https://github.com/geode-sdk/geode/assets/94473358/822aa16c-01f8-4c60-8c40-3268f83ad0f0)
this seems to be because Geode is built as i386 (32bit) while GD is now x86-64 (64bit)
